### PR TITLE
 fix Dom nesting error in singleFIMap

### LINKED ISSUE #366: Error on FI map page
```sh
index.js:1437 Warning: validateDOMNesting(...): <h5> cannot appear as a child of <h2>.
    in h5 (at active/index.tsx:309)
    in h2 (at active/index.tsx:307)
    in div (created by Col)
    in Col (at active/index.tsx:306)
    in div (created by Row)
    in Row (at active/index.tsx:305)
    in div (at active/index.tsx:304)
    in div (at active/index.tsx:299)
    in SingleActiveFIMap (created by Context.Consumer)
    in Connect(SingleActiveFIMap) (created by Route)
    in Route (created by PrivateRoute)
    in PrivateRoute (created by Context.Consumer)
    in Connect(PrivateRoute) (at App.tsx:91)
    in Switch (at App.tsx:56)
    in div (created by Col)
    in Col (at App.tsx:55)
    in div (created by Row)
    in Row (at App.tsx:54)
    in div (created by Container)
    in Container (at App.tsx:51)
    in App (at src/index.tsx:15)
    in Router (created by ConnectedRouter)
    in ConnectedRouter (created by Context.Consumer)
    in ConnectedRouterWithContext (created by Context.Consumer)
    in Connect(ConnectedRouterWithContext) (at src/index.tsx:14)
    in Provider (at src/index.tsx:13)
```

### PR DESCRIPTION
**Pull request by p-netm**
replace heading tag that is semantically not-well nested with a p tag
 add heading utility class for appropriate styling.

closes #366

### CODE REVIEW

File: src/containers/pages/FocusInvestigation/map/active/index.tsx [hunk 1]
```diff
@@ -331,10 +333,10 @@ class SingleActiveFIMap extends React.Component<
           <div className="col-3">
             <div className="mapSidebar">
               <div>
-                <h5>
+                <h1 className="h5">
```
Comment (moshthepitt) [inline src/containers/pages/FocusInvestigation/map/active/index.tsx]: This is a somewhat odd class name for a h1 element, no?

Comment (cKellyDesign) [reply src/containers/pages/FocusInvestigation/map/active/index.tsx]: Agreed, this should either be `h3` or remain `h5`. If this were in an actual html5 `aside` element then that would be semantically different, but as is shouldn't be `h1`.

Comment (p-netm) [reply src/containers/pages/FocusInvestigation/map/active/index.tsx]: I think the other option would be to use the `p` tag and add the className of h5

Comment (p-netm) [reply src/containers/pages/FocusInvestigation/map/active/index.tsx]: just found I actually did not need to make changes to that line.

File: src/containers/pages/FocusInvestigation/map/active/index.tsx [hunk 2]
```diff
@@ -306,7 +306,9 @@ class SingleActiveFIMap extends React.Component<
             <Col xs="8">
               <h2 className="page-title mt-4 mb-4">
                 {plan && plan.plan_title} {INVESTIGATION}{' '}
-                <h5 style={{ display: 'inline' }}>{statusBadge}</h5>
+                <p className="h5" style={{ display: 'inline' }}>
```
Comment (moshthepitt) [inline src/containers/pages/FocusInvestigation/map/active/index.tsx]: I am not feeling great as having "h5" as the CSS class.  It seems an odd choice to me

Comment (p-netm) [reply src/containers/pages/FocusInvestigation/map/active/index.tsx]: Alternatives:
- Using a custom css class to make the `p` look like h5.
- use a h3 for proper nesting:
    - leave the default styling for the h3
    - write a custom style class to make it look like h5 .
- others?
I think the domElement in question has to be a `h3` or any other text non-heading domElement.

Comment (moshthepitt) [reply src/containers/pages/FocusInvestigation/map/active/index.tsx]: My question was more regarding the choice of name "h5" i.e. it feels weird to have it named as the name of another html element

Comment (p-netm) [reply src/containers/pages/FocusInvestigation/map/active/index.tsx]: the `h5` is from bootstrap.

Comment (moshthepitt) [reply src/containers/pages/FocusInvestigation/map/active/index.tsx]: aaah, okay

**Pull request merged**